CMakeLists.txt

```diff
@@ -9,6 +9,8 @@
 PROJECT(lua-archive C)
 CMAKE_MINIMUM_REQUIRED (VERSION 2.6)
 
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -O0 -g3 -ggdb")
```

Comment:
Perhaps this should be reverted since production installs will probably not want this setting? ...or at least comment it out?